src/components/token/TokenBalances.js

```diff
@@ -315,7 +335,9 @@ class TokenBalances extends React.Component {
             !this.state.tokenBalanceInformationError && (
               <>
                 <p><b>Total number of addresses:</b> { helpers.renderValue(this.state.addressesCount, true) }</p>
-                <p><b>Total number of transactions:</b> { helpers.renderValue(this.state.transactionsCount, true) }</p>
+                {!this.state.tokensApiError && (
```

Comment:
Should we write any error message here? So we can let the user know that he can reload the scree to try to get this information.

Reply:
I agree, done in 180368e84ebd62af974b4f73f28d639ee0d606dd